File: Seccion 11 - Funciones en C++/Seccion_11_Dart/06_ejercicio_04.dart

import 'dart:io';

double numero;

void pedirDatos() {
  stdout.writeln('Digite un número de punto flotante: ');
  numero = double.parse(stdin.readLineSync());
}

double parteFraccionaria(double r) {
  int entero = r.toInt();

  double resultado = r - entero;

  return resultado;
}

void main() {

  pedirDatos();
  print('LA parte fraccionaria del número $numero es: ${parteFraccionaria(numero)}');
}

File: Seccion 11 - Funciones en C++/Seccion_11_Python/06_ejercicio_04.py

numero = 0.0

def pedir_datos():
    global numero
    numero = float(input('Digite un número de punto flotante: '))

def parte_fraccionaria(r):
    entero = int(r)

    resultado = r - entero

    return resultado

pedir_datos()
print('La parte fraccionaria del número {} es: {}'.format(numero, parte_fraccionaria(numero)))